full-stack-tests/core/src/frontend/app/NativeApp.test.ts

```diff
@@ -12,7 +12,7 @@ import { TestUtility } from "../TestUtility";
 
 if (ProcessDetector.isElectronAppFrontend) {
 
-  describe("NativeApp startup", () => {
+  describe.skip("NativeApp startup", () => {

```

Comment:
@khanaffan If this test is run, subsequent tests fail because of the issue mentioned in FIXME.
Do we intend to fix?
```
  NativeApp startup
    ? should startup offline without errors (11275ms)

  NativeApp Storage
    ? Primitive Types (70ms)
    ? Override and type check (81ms)

  ViewState attached to Viewport
Uncaught Error in Tests:  Cannot read properties of undefined (reading 'notifyResponse')
TypeError: Cannot read properties of undefined (reading 'notifyResponse')
    at FrontendIpcTransport.handleComplete (D:/js/s/imodeljs/core/electron/src/common/ElectronIpcTransport.ts:161:13)
    at EventEmitter.<anonymous> (D:/js/s/imodeljs/core/electron/src/common/ElectronIpcTransport.ts:87:14)
    at EventEmitter.emit (events.js:376:20)
    at Object.onMessage (electron/js2c/renderer_init.js:81:1017)
```

Reply:
@calebmshafer you added these changes in commit f45f1db6827b693c3e9b9097b18741169265f2f9.

Reply:
https://github.com/iTwin/itwinjs-core/issues/4165